packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -774,21 +760,52 @@ export class ContainerRuntime
 
 		const mc = loggerToMonitoringContext(logger);
 
+		// Some options require a minimum version of the FF runtime to operate, so the default configs will be generated
+		// based on the compatibility mode.
+		// For example, if compatibility mode is set to "1", the default configs will ensure compatibility with FF runtime
+		// 1.x  or later. If the compatibility mode is set to "2", the default values will be generated to ensure compatibility
+		// with FF runtime 2.x or later.
+		// Our policy is to support N/N-1 compatibility by default, where N is the most recent public major release of the runtime.
+		// Therefore, if the customer does not provide a compatibility mode, we will default to use N-1.
+		const defaultCompatibilityMode = "1"; // TODO: Automatically set to be N-1.
```

Comment:
As mentioned elsewhere, can we have the default be "2"? (always set to N.0.0)

Reply:
Ok. I don't know who wrote that policy or how binding it is.  How about it kicks in starting in 3.0, but for 2.0 the default is what it currently is... which is only N/N support.

Reply:
In the latest commit I try out having the default compat mode be 2.0. This raises some issues with e2e tests, but those will likely be addressed in the next phase of compat work anyway.